README.md

```diff
@@ -151,9 +157,13 @@ $ sudo apt-get install android-tools-adb android-tools-fastboot autoconf \
 	mtools netcat python-crypto python3-crypto python-pyelftools \
 	python3-pycryptodome python3-pyelftools python-serial python3-serial \
 	rsync unzip uuid-dev xdg-utils xterm xz-utils zlib1g-dev
+```
 
+After installing dependencies or building the Docker container, fetch the source code from the official GitHub repository:
```

Comment:
Please wrap the long lines at 80 characters.
Reference:  https://vim.fandom.com/wiki/Format_only_long_lines